src/blog/kuadrant-roadmap/kuadrant-roadmap.md

```diff
@@ -0,0 +1,110 @@
+Here's an example of a feature [filtered view of the backlog for Azure Support](https://github.com/orgs/Kuadrant/projects/18/views/22?sliceBy%5BcolumnId%5D=73323580&sliceBy%5Bvalue%5D=Azure+Support).
+
+*Where does this leave Epics?*
+
+    Still valuable ….
+
+### Iterations
+
+An iteration is GH is the name given to identify a period of time. It is a useful tool for understanding what is happening within a given period of time and also what we have committed to and planned and will ideally make it in the next iteration (or release) of the project.
+Why use an iteration?
+
+* Allows a shared timebox for the next iteration of the project. Everyone is working with the same iteration to the same date and knows there will be a release at the end of the iteration.
+* Improves visibility into what is happening within a given time frame
+* Breaks down a large period of time (months until November) into smaller easier chunks that us humans can get our heads around.
+* We work in 3-week interactions. This gives us a total of 5 iterations between now and KubeCon NA 2024
+
+### Estimates
+
+* Estimates are exactly that. No one will come chasing you if you get an estimate wrong.
+* Estimates will help us gauge how much we can complete in a given iteration and also in a given large period of time. 
+* Suggested Estimate Values to keep things simple:
+  * 5:  1-2 days effort
+  * 8:  3-5 days effort
+  * 13: anything from 5 days to 2 weeks effort
+* If an issue is larger than a 13 and can’t be completed within an iteration, then really it should be looked at closely to see if it can be broken down further. A 13 point issue should also probably be looked carefully to see can it be broken down further
+
+### Releases
+
+While we release an increment with the end of each iteration. The release field (should prob consider renaming) is more of a milestone release. e.g v1. This release version should be added to any work that is a “must do” for v1. 
+
+## Using The Views
+
+### Adding New Stuff
+
+#### Areas
+
+If a new area is discovered, it can be discussed on Slack or community call as to why it is big enough for a new area all of its own. That said it won't be a heavy-weight process to add a new area.
+
+#### Features
+
+Expectation is that these will naturally occur as work is planned and outlined. It is just a string so there is no real limitation on it.
+
+#### Adding New Issues
+
+It's important that the right area, feature and version is assigned. This is just a suggestion. If you are adding a new issue, it can be helpful to go to the feature option on the view and then click the “add new issue” at the bottom of the list as this will auto fill the feature and area. I
```

Comment:
```suggestion
It's important that the right area, feature and version is assigned. This is just a suggestion. If you are adding a new issue, it can be helpful to go to the feature option on the view and then click the “add new issue” at the bottom of the list as this will auto fill the feature and area.
```